devices/templatetags/devicetags.py

```diff
@@ -129,3 +129,15 @@ def as_nested_list(factvalue):
 @register.filter
 def splitstr(arg1, arg2):
     return arg1.split(arg2)
+
+
+@register.filter
+def addstr(arg1, arg2):
```

Comment:
this not supposed to be part of the PR, right?

Reply:
Yes it is, I wanted a template tag that doesn't change behavior when given a string that can be cast to an int and as I first needed it in this pr, I thought it belonged in this pr.